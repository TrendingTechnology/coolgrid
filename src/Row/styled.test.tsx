import React from 'react'
import { shallow } from 'enzyme'
import Component from './styled'

describe('Row - styled component', () => {
  it('renders correctly', async () => {
    const tree = shallow(<Component />)
    expect(tree).toMatchSnapshot()
  })
})
