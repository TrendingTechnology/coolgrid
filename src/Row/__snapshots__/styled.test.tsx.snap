// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row - styled component renders correctly 1`] = `
<div
  className="sc-bdVaJa kJujun"
/>
`;
